pkg/vex/statement.go

```diff
@@ -34,9 +53,8 @@ type Statement struct {
 
 	// For "affected" status, a VEX statement MUST include an ActionStatement that
 	// SHOULD describe actions to remediate or mitigate [vul_id].
-	ActionStatement string `json:"action_statement,omitempty"`
-
-	References []VulnerabilityReference `json:"references,omitempty"` // Optional list
+	ActionStatement          string    `json:"action_statement,omitempty"`
+	ActionStatementTimeStamp time.Time `json:"action_statement_timestamp,omitempty"`
```

Comment:
nit: lowercase "s" in "Timestamp" for consistency?

```suggestion
	ActionStatementTimestamp time.Time `json:"action_statement_timestamp,omitempty"`
```

Reply:
Also in looking at the data model doc comments, I'm wondering if this is going to be removed in favor of statement timestamps? I may be missing something. I'm trying to wrap my head around how this would value would be decided, and then how VEX consumers would use it.

Reply:
IIRC this one was a separate timestamp from the statement ones. Should we hold on adding this one until it is finalized?

Reply:
Okay yeah, I think that's a good approach

Reply:
OK, I piggybacked the `ActionStatementTimeStamp` case into the next commit.